src/components/common/Loader.js

```diff
@@ -9,14 +10,39 @@ const useStyles = makeStyles((theme) => ({
     flex: 1,
     margin: theme.spacing(4),
   },
+  '@keyframes blinker': {
+    from: { opacity: 1 },
+    to: { opacity: 0 },
+  },
+  additionalText: {
+    marginTop: theme.spacing(4),
+    animation: '$blinker',
+    animationDuration: '2s',
+    animationTimingFunction: 'linear',
+    animationIterationCount: 'infinite',
+  },
 }));
 
 const Loader = () => {
   const classes = useStyles();
+  const [showAdditionalText, setShowAdditionalText] = useState(false);
+
+  // after 5 seconds, display 'still loading' text by changing state of showAdditionalText
+  useEffect(() => {
+    setTimeout(() => {
+      setShowAdditionalText(true);
+    }, 5000);
+  }, []);
+
   return (
     <Grid container>
       <Grid item className={classes.gridItem}>
         <CircularProgress />
+        {showAdditionalText ? (
```

Comment:
`showAdditionalText && ` to avoid the ternary.

Reply:
100%